Packs/MicrosoftGraphGroups/ReleaseNotes/1_0_13.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Active Directory Groups
+- Added the *Key* integration parameter to support credentials fetching object.
```

Comment:
```suggestion
- Added the *Key* integration parameter to support credentials fetching object.
  ***Note:*** if an integration instance is already configured, no action is required.
```
@Itay4 correct me if I'm wrong, worth mentioning

Reply:
i think its a bit misleading as it might suggest that without action the user can now use the credential object
i would leave it is without this addition